README.md

```diff
@@ -18,7 +18,11 @@ NOTE: This has a peer dependencies of `rxjs@5.0.*` and `redux`, which will have
 as well.
 
 ```sh
-npm install --save redux-observable
```

Comment:
This is instructions on how to install redux-observable into your own project, not how to install the and run the examples.